mlx/backend/metal/metal.cpp

```diff
@@ -88,20 +90,26 @@ std::function<void()> make_task(
     if (!arr.is_tracer()) {
       arr.detach();
     }
-    if (p) {
+
+    if (signal) {
+      // Make sure this buffer had at least one command encoder on it.
```

Comment:
Is there something I am missing for this comment?

Reply:
No, sorry from a previous bug